blockproducer/rpc.go

```diff
@@ -139,6 +134,18 @@ func (s *ChainRPCService) QuerySQLChainProfile(req *types.QuerySQLChainProfileRe
 	return
 }
 
+func (s *ChainRPCService) QueryTxState(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method ChainRPCService.QueryTxState should have comment or be unexported